packages/components/tests/integration/components/hds/form/sensitive-input/field-test.js

```diff
@@ -0,0 +1,146 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render, resetOnerror } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+module(
+  'Integration | Component | hds/form/sensitive-input/field',
+  function (hooks) {
+    setupRenderingTest(hooks);
+
+    hooks.afterEach(() => {
+      resetOnerror();
+    });
+
+    test('it should render the component with a specific CSS class', async function (assert) {
+      await render(hbs`<Hds::Form::SensitiveInput::Field />`);
+      assert.dom('.hds-form-field__control').exists();
+    });
+
+    // VALUE
+
+    test('it should render the input with the value provided via @value argument', async function (assert) {
+      await render(hbs`<Hds::Form::SensitiveInput::Field @value="abc123" />`);
+      assert.dom('input').hasValue('abc123');
+    });
+
+    // INVALID
+
+    test('it should render the correct CSS class if the @isInvalid prop is declared', async function (assert) {
+      await render(
+        hbs`<Hds::Form::SensitiveInput::Field @isInvalid={{true}} />`
+      );
+      assert.dom('input').hasClass('hds-form-text-input--is-invalid');
+    });
+
+    // WIDTH
+
+    test('it should render the input with a fixed width if a @width value is passed', async function (assert) {
```

Comment:
should we test also the height for the multiline version?

Reply:
Added in [6361cd8](https://github.com/hashicorp/design-system/pull/1468/commits/6361cd83e7bf6fd349a1df8a1fbce220319a109a)